### DISCUSSION TITLE
Is there a view that has the request url, headers, payload

### DISCUSSION THREAD

**Discussion by LintaoAmons** [mistweaverco/kulala.nvim#301] [Q&A] [answered]
Just like the `network` panel inside dev tool of chrome.

**Comment by gorillamoe** [comment]
No, there is no such view. I don't see any real benefit to this as long as the responses are not stored and reviewed later as in the feature request #292.

We might consider this, when #292 may be implemented.

**Comment by gorillamoe** [comment, marked as answer]
Implemented in verbose mode by @YaroSpace .
